nens_auth_client/models.py

```diff
@@ -233,17 +233,28 @@ def send_email(self, request, context=None, send_email_options=None):
             "host": request.get_host(),
             **(context or {}),
         }
-
-        text = render_to_string("nens_auth_client/invitation.txt", context=context)
-        html = render_to_string("nens_auth_client/invitation.html", context=context)
+        if context.get("service_email", None) is None:
```

Comment:
`get("service_email", None)` kan gewoon `get("service_email")` worden, want None is de default.